addons/account/models/account_move_line.py

```diff
@@ -3222,12 +3222,12 @@ def _prepare_edi_vals_to_export(self):
         return res
 
     @api.model
-    def read_group(self, domain, fields, groupby, offset=0, limit=None, orderby=False, lazy=True):
-        # Hide total amount_currency from read_group when view is not grouped by currency_id. Avoids mix of currencies
-        res = super().read_group(domain, fields, groupby, offset=offset, limit=limit, orderby=orderby, lazy=lazy)
-        if 'currency_id' not in groupby and 'amount_currency:sum' in fields:
+    def formatted_read_group(self, domain, groupby=(), aggregates=(), having=(), offset=0, limit=None, order=None) -> list[dict]:
```

Comment:
Note that in theory since https://github.com/odoo/odoo/commit/e96566e11d6084c6f510361ff1e5cb243eb3a84e we shouldn't need this anymore